vm/hash.go

```diff
@@ -127,6 +167,18 @@ var builtinHashInstanceMethods = []*BuiltInMethodObject{
 		},
 	},
 	{
+		// Associates the value given by `value` with the key given by `key`.
+		// Returns the `value`.
+		//
+		// ```Ruby
+		// h = { a: 1, b: "2", c: [1, 2, 3], d: { k: 'v' } }
+		// h['a'] = 1          #=> 1
+		// h['b'] = "2"        #=> "2"
+		// h['c'] = [1, 2, 3]  #=> [1, 2, 3]
+		// h['d'] = { k: 'v' } #=> { k: 'v' }
+		// ```
+		//
+		// @return [value]
```

Comment:
The name in the brackets is the class of the returned value, so it should be `[Object]` instead. The `value` can go to explanation afterwards.